timestamp-batch/pom.xml

```diff
@@ -65,6 +71,13 @@
 
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
+				<groupId>com.oracle.database.jdbc</groupId>
+				<artifactId>ojdbc-bom</artifactId>
+				<version>${oracle-jdbc.version}</version>
+				<type>pom</type>
```

Comment:
So the BOM should only guide us on versions and this will not bring in all the things.  After `mvn package` I would expect to see the `com.oracle.database.jdbc:ojdbc8:<version-from-bom>` in the `BOOT-INF/lib`. Do you mind verifying that (if you did not already)? Thanks..

Reply:
That's what should happen

Reply:
jar tvf timestamp-task-2.0.1.jar | grep ojdbc
5040465 Fri Oct 09 09:34:58 EDT 2020 BOOT-INF/lib/ojdbc8-21.1.0.0.jar